README.md

```diff
@@ -150,6 +150,26 @@ For further details of how to construct a language driver,
 take a look at [Implementing the driver](https://doc.bblf.sh/driver/sdk.html#implementing-the-driver)
 section in documentation.
 
+### Testing the driver
+
+`bbflsh-sdk` also includes a testing framework for a driver.
+In order to run test for a particular dirver, change to it's directory and run:
+
+```bash
+$ bblfsh-sdk test
+```
+
+This will:
+ - compile a "test binary" that parses content of the `./fixtures` directory of the driver
+ - create a docker image with all dependencies, native driver and a test binary
+ - run this test binary inside a Docker container, using that image
+
+ Overall, SDK supports 3 different kind of tests for a driver:
+  - UnitTests, parsing content of `./fixtures` and applying UAST transformations. Described above.
+  - Integration tests, using content of `./fixtures/_integration*`
+  - Benchmarks, using content of `./fixtures/bench_*`
```

Comment:
There are also unified tests, the ones starting with `u2_` which implement the same things with a unified struct. Used to test semantic objects.

Reply:
So far I only documented the things I know and understand :) 

If you could provide an exact wording here - I'll be happy to add! 

Otherwise will keep it as is and update it later, when I stumble up on `u2_` and understand their purpose better.